src/models/analysis.js

```diff
@@ -82,6 +82,11 @@ Analysis.getAnalysisWithDetails = (ajaxGet = simpleGet) =>
           token: props.token,
           actorId: analysis.registeredBy
         }).map(actor => actor ? { ...analysis, registeredByName: actor.fn } : analysis))
+      .flatMap(analysis =>
+        MusitActor.getActor(ajaxGet)({
+          token: props.token,
+          actorId: analysis.updatedBy
+        }).map(actor => actor ? { ...analysis, updatedByName: actor.fn } : analysis))
```

Comment:
just a question, have you run "npm run formatcode"? there is a dangling ")" here, which is suppose should be on the next line, so I assume formatting has not been done. I also see an extra line AnalysisAddComponent, so the theory that the files are not formatted is a strong theory.